### ISSUE TITLE
Crash on Authentication

### ISSUE THREAD

**Issue by timendez** [playgameservices/play-games-plugin-for-unity#2627]
Hi, I've been racking my brain about this crash for the past couple of days.

My game [Silly Slots](https://play.google.com/store/apps/details?id=com.TImMendez.SillySlots&hl=en_US) has been working with the Play Games Plugin for Unity since I released it. I recently decided to update the game after not touching the code for 2 years. At first I updated the project to my current version of Unity (2018.3). I was able to get the project to run fine in the editor, IAPs, Unity Ads both work fine. Commenting out the line:

```
Social.localUser.Authenticate(...
```
causes the game to behave as normal, except then the user is not authenticated to GPGS.

I've attempted the `\u003` fix, updated Unity to the latest LTS version (2019.1.9f1 Personal), updated to the latest play-games-plugin-for-unity (0.9.64), updated to the latest play services resolver (v1.2.122), re-set up GPG.

This might be a keystore problem - I copied my original production keystore over from my old machine and tried using that and it built fine, but perhaps it still is a signing problem because I'm not too well versed in that area.

This is also built on a new machine running MacOS Mojave whereas before it was on High Sierra but I doubt that's what the problem is.

```
07-10 13:54:46.152  8954  8982 I Unity   : Starting Auth with token client.
07-10 13:54:46.152  8954  8982 I Unity   :  
07-10 13:54:46.152  8954  8982 I Unity   : (Filename: ./Runtime/Export/Debug/Debug.bindings.h Line: 48)
07-10 13:54:46.152  8954  8982 I Unity   : 
07-10 13:54:46.365  8954  8982 I Unity   :  [Play Games Plugin DLL] 07/10/19 13:54:46 -07:00 DEBUG: Activating PlayGamesPlatform.
07-10 13:54:46.365  8954  8982 I Unity   :  
07-10 13:54:46.365  8954  8982 I Unity   : (Filename: ./Runtime/Export/Debug/Debug.bindings.h Line: 48)
07-10 13:54:46.365  8954  8982 I Unity   : 
07-10 13:54:46.365  8954  8982 I Unity   :  [Play Games Plugin DLL] 07/10/19 13:54:46 -07:00 DEBUG: Instance was not initialized, using default configuration.
07-10 13:54:46.365  8954  8982 I Unity   :  
07-10 13:54:46.365  8954  8982 I Unity   : (Filename: ./Runtime/Export/Debug/Debug.bindings.h Line: 48)
07-10 13:54:46.365  8954  8982 I Unity   : 
07-10 13:54:46.367  8954  8982 W Unity   : !!! [Play Games Plugin DLL] 07/10/19 13:54:46 -07:00 WARNING: Creating new PlayGamesPlatform
07-10 13:54:46.367  8954  8982 W Unity   :  
07-10 13:54:46.367  8954  8982 W Unity   : (Filename: ./Runtime/Export/Debug/Debug.bindings.h Line: 48)
07-10 13:54:46.367  8954  8982 W Unity   : 
07-10 13:54:46.367  8954  8982 I Unity   :  [Play Games Plugin DLL] 07/10/19 13:54:46 -07:00 DEBUG: PlayGamesPlatform activated: GooglePlayGames.PlayGamesPlatform
07-10 13:54:46.367  8954  8982 I Unity   :  
07-10 13:54:46.367  8954  8982 I Unity   : (Filename: ./Runtime/Export/Debug/Debug.bindings.h Line: 48)
07-10 13:54:46.367  8954  8982 I Unity   : 
07-10 13:54:46.367  8954  8982 I Unity   :  [Play Games Plugin DLL] 07/10/19 13:54:46 -07:00 DEBUG: Creating platform-specific Play Games client.
07-10 13:54:46.367  8954  8982 I Unity   :  
07-10 13:54:46.367  8954  8982 I Unity   : (Filename: ./Runtime/Export/Debug/Debug.bindings.h Line: 48)
07-10 13:54:46.367  8954  8982 I Unity   : 
07-10 13:54:46.367  8954  8982 I Unity   :  [Play Games Plugin DLL] 07/10/19 13:54:46 -07:00 DEBUG: Creating Android IPlayGamesClient Client
07-10 13:54:46.367  8954  8982 I Unity   :  
07-10 13:54:46.367  8954  8982 I Unity   : (Filename: ./Runtime/Export/Debug/Debug.bindings.h Line: 48)
07-10 13:54:46.367  8954  8982 I Unity   : 
07-10 13:54:46.628  1543  3073 I ActivityManager: START u0 {act=com.google.android.gms.auth.GOOGLE_SIGN_IN pkg=com.TImMendez.SillySlots cmp=com.TImMendez.SillySlots/com.google.android.gms.auth.api.signin.internal.SignInHubActivity (has extras)} from uid 10413
07-10 13:54:47.049  1543  3892 I ActivityManager: START u0 {act=com.google.android.gms.auth.GOOGLE_SIGN_IN pkg=com.google.android.gms cmp=com.google.android.gms/.auth.api.signin.ui.SignInActivity (has extras)} from uid 10413
07-10 13:54:47.362  1543  3088 W ActivityManager: Sending non-protected broadcast com.lge.systemui.notifyImmersiveModeType from system 2567:com.android.systemui/u0a121 pkg com.android.systemui
07-10 13:54:47.412  1543  1670 I ActivityManager: Displayed com.google.android.gms/.auth.api.signin.ui.SignInActivity: +277ms (total +384ms)
07-10 13:54:47.489  1543  4527 I ActivityManager: START u0 {cmp=com.google.android.gms/.common.api.GoogleApiActivity (has extras)} from uid 10079
07-10 13:54:47.548  1543  3088 I ActivityManager: START u0 {act=com.google.android.gms.games.SIGN_IN flg=0x4000000 pkg=com.google.android.play.games cmp=com.google.android.play.games/com.google.android.gms.games.ui.signin.SignInActivity (has extras)} from uid 10079

07-10 13:54:48.217  8954  8954 I Unity   : Building GPG services, implicitly attempts silent auth
07-10 13:54:48.217  8954  8954 I Unity   :  
07-10 13:54:48.217  8954  8954 I Unity   : (Filename: ./Runtime/Export/Debug/Debug.bindings.h Line: 48)
07-10 13:54:48.217  8954  8954 I Unity   : 
07-10 13:54:48.375  1543  4526 W ActivityManager: Sending non-protected broadcast com.lge.systemui.notifyImmersiveModeType from system 2567:com.android.systemui/u0a121 pkg com.android.systemui
07-10 13:54:48.376  8954  8982 I Unity   : user authenticated = False
07-10 13:54:48.376  8954  8982 I Unity   :  
07-10 13:54:48.376  8954  8982 I Unity   : (Filename: ./Runtime/Export/Debug/Debug.bindings.h Line: 48)
07-10 13:54:48.376  8954  8982 I Unity   : 
07-10 13:54:48.384  8954  8982 I Unity   :  [Play Games Plugin DLL] 07/10/19 13:54:48 -07:00 DEBUG: Invoking user callback on game thread
07-10 13:54:48.384  8954  8982 I Unity   :  
07-10 13:54:48.384  8954  8982 I Unity   : (Filename: ./Runtime/Export/Debug/Debug.bindings.h Line: 48)
07-10 13:54:48.384  8954  8982 I Unity   : 
07-10 13:54:48.436  1543  4526 W ActivityManager:   Force finishing activity com.TImMendez.SillySlots/com.unity3d.player.UnityPlayerActivity
07-10 13:54:48.443  1543  1627 I ActivityManager: Showing crash dialog for package com.TImMendez.SillySlots u0
07-10 13:54:48.484  1543  3073 W ActivityManager: Sending non-protected broadcast com.lge.mlt.action.smartcare.event from system 3751:com.lge.mlt/1000 pkg com.lge.mlt
07-10 13:54:48.569  1543  1627 I ActivityManager: Config changes=480 {1.0 310mcc410mnc [en_US] ldltr sw360dp w360dp h644dp 640dpi nrml long hdr port finger -keyb/v/h -nav/h appBounds=Rect(0, 0 - 1440, 2672) s.99 fontTypeIndex0 forceEmbolden0}
07-10 13:54:48.666  1543  1627 I ActivityManager: Override config changes=480 {1.0 310mcc410mnc [en_US] ldltr sw360dp w360dp h644dp 640dpi nrml long hdr port finger -keyb/v/h -nav/h appBounds=Rect(0, 0 - 1440, 2672) s.99 fontTypeIndex0 forceEmbolden0} for displayId=0
07-10 13:54:48.678  1543  1627 D ActivityManager: update defaultMinSize
07-10 13:54:48.678  1543  1627 D ActivityManager: calculateDefaultMinimalSizeOfResizeableTasks from 880 to 880
07-10 13:54:48.938  1543  1626 W ActivityManager: Activity pause timeout for ActivityRecord{245e807 u0 com.TImMendez.SillySlots/com.unity3d.player.UnityPlayerActivity t1509 f}
07-10 13:54:49.368  1543  8381 W ActivityManager: Sending non-protected broadcast com.lge.systemui.notifyImmersiveModeType from system 2567:com.android.systemui/u0a121 pkg com.android.systemui
07-10 13:54:50.594  1543  4526 W ActivityManager:   Force finishing activity com.TImMendez.SillySlots/com.unity3d.player.UnityPlayerActivity
07-10 13:54:50.614  8954  8982 E Unity   : [EGL] Failed to create window surface: EGL_BAD_ALLOC: EGL failed to allocate resources for the requested operation.
07-10 13:54:50.614  8954  8982 E Unity   :  
07-10 13:54:50.614  8954  8982 E Unity   : (Filename: ./Runtime/GfxDevice/egl/WindowContextEGL.cpp Line: 161)
07-10 13:54:50.614  8954  8982 E Unity   : 
07-10 13:54:50.614  8954  8982 I Unity   : Skipped rendering frame because GfxDevice is in invalid state (device lost)
07-10 13:54:50.614  8954  8982 I Unity   :  
07-10 13:54:50.614  8954  8982 I Unity   : (Filename: ./Runtime/Misc/Player.cpp Line: 782)
07-10 13:54:50.614  8954  8982 I Unity   : 
07-10 13:54:50.616  8954  8982 E Unity   : [EGL] Failed to create window surface: EGL_BAD_ALLOC: EGL failed to allocate resources for the requested operation.
07-10 13:54:50.616  8954  8982 E Unity   :  
07-10 13:54:50.616  8954  8982 E Unity   : (Filename: ./Runtime/GfxDevice/egl/WindowContextEGL.cpp Line: 161)
07-10 13:54:50.616  8954  8982 E Unity   : 
07-10 13:54:50.616  8954  8982 I Unity   : Skipped rendering frame because GfxDevice is in invalid state (device lost)
07-10 13:54:50.616  8954  8982 I Unity   :  
07-10 13:54:50.616  8954  8982 I Unity   : (Filename: ./Runtime/Misc/Player.cpp Line: 782)
07-10 13:54:50.616  8954  8982 I Unity   : 
07-10 13:54:50.620  1543  4526 I ActivityManager: Killing 8954:com.TImMendez.SillySlots/u0a413 (adj 900): crash
```

Any help is greatly appreciated, let me know if you need anything else - thanks!

**Comment by DimoNline** [comment]
Same, did you solved this?

**Comment by legas1** [comment]
Can you please post full logcat log (upload it to pastebin or similar site)?

**Comment by olehkuznetsov** [comment]
Should be fixed in v0.10.06. Feel free to reopen if it reproduces there.
